src/ALZ/Private/New-ALZDirectoryEnvironment.ps1

```diff
@@ -10,7 +10,7 @@ function New-ALZDirectoryEnvironment {
     # Create destination file structure
     $gitHubPipeline = Join-Path $alzEnvironmentDestination ".github" "workflows"
     $config = Join-Path $alzEnvironmentDestination "config"
-    $configModules = Join-Path $alzEnvironmentDestination "config" "modules"
+    $configModules = Join-Path $alzEnvironmentDestination "config" "custom-modules"
```

Comment:
See how quickly I lose context after a few days away :D  This is adopting a directory name change in ALZ-Bicep?